deploy-manage/deploy/cloud-on-k8s/configuration-examples-standalone.md

```diff
@@ -47,9 +47,23 @@ Deploys two {{es}} clusters and two {{kib}} instances together with single Elast
 
 ## Storing local state in host path volume [k8s_storing_local_state_in_host_path_volume]
 
-{{agent}} managed by ECK stores local state in a host path volume by default. This ensures that {{integrations}} run by the agent can continue their work without duplicating work that has already been done after the Pod has been recreated for example because of a Pod configuration change. Multiple replicas of an agent, for example {{fleet}} Servers, can not be deployed on the same underlying {{k8s}} node as they would try to use the same host path. There are 2 options for managing this feature:
+{{agent}} managed by ECK stores local state in a host path volume by default. This ensures that {{integrations}} run by the agent can continue their work without duplicating work that has already been done after the Pod has been recreated for example because of a Pod configuration change. Multiple replicas of an agent, for example {{fleet}} Servers, can not be deployed on the same underlying {{k8s}} node as they would try to use the same host path. There are 1 or 2 options for managing this feature depending on the version of Elastic Agent.
+
+::::{applies-switch}
+
+:::{applies-item} { "stack": "ga 8.16" } In Elastic Agent versions 8.16 and beyond:
```

Comment:
looked again at this - this might be failing because applies-item doesn't accept freeform text - just version info

because this is stack 8.x, consider just using a regular tabset and just the titles - 

```
::::{tab-set}

:::{tab-item} {{agent}} 8.16 and higher
```	

we are waiting for some functionality in the applies tags to better indicate ranges that will let you use applies switch more readily here in future